src/widgets/mentors-register/ui/mentors-register.module.scss

```diff
@@ -22,6 +27,7 @@
 
   @include media-tablet {
     max-width: 290px;
+    max-height: 272px;
```

Comment:
With such a width, such `max-height` will distort the proportions

Reply:
Done